app/src/androidTest/kotlin/com/adriangl/pokeapi_mvvm/fixtures/MoveFixture.kt

```diff
@@ -0,0 +1,54 @@
+package com.adriangl.pokeapi_mvvm.fixtures
+
+import com.adriangl.pokeapi_mvvm.network.Generation
+import com.adriangl.pokeapi_mvvm.network.MoveType
+import com.adriangl.pokeapi_mvvm.network.PokemonMove
+import com.adriangl.pokeapi_mvvm.network.Target
+
+
+interface MoveFixtures {
+
+    private val baseMove
+        get() = PokemonMove(
```

Comment:
Do you need the `get()` here?

Reply:
Yep, you cannot instance a variable inside an interface

Reply:
`property initializers are not allowed in Interfaces`